frappe/desk/page/setup_wizard/setup_wizard.py

```diff
@@ -54,9 +54,17 @@ def setup_complete(args):
 		return {'status': 'ok'}
 
 	args = parse_args(args)
-
 	stages = get_setup_stages(args)
+	is_background_task = frappe.conf.get('trigger_site_setup_in_background')
```

Comment:
```suggestion
	is_background_task = frappe.conf.setup_wizard_priority == "background"
```
How about a different name?

---------

Or, would it be possible to handle this directly in ERPNext? Maybe we can have individual stages in the setup wizard that can be marked as background tasks? 😅

Reply:
`setup_wizard_priority`  is better than `trigger_site_setup_in_background`. But I prefer bool config over string value (easy to configure and no need to remember what value to use). Let me know if you have any other names in mind..

Reply:
> `setup_wizard_priority` is better than `trigger_site_setup_in_background`. But I prefer bool config over string value (easy to configure and no need to remember what value to use). Let me know if you have any other names in mind..

Yes, doable.. But I guess steps are dependent on one another and that might be the reason for not moved them into background. I assume this can be done even with current setup.

Reply:
> setup_wizard_priority is better than trigger_site_setup_in_background. But I prefer bool config over string value (easy to configure and no need to remember what value to use).

How about `setup_wizard_priority` only, but keep it boolean/numeric. Falsy can mean in-request/foreground and Truthy can mean background? Or we can have the other way around too.

Reply:
I prefer names to be self explained. I think we better merge as it is for now. You can change when ever you come up with better name. Hope that works!